File: javaFiles/Inheritence.java
package javaFiles;

public class Inheritence {
    String category = "Roller Coasters";
    public void startRide() {
        System.out.println("The ride has started!");
    }
}

class ThemeParkRide extends Inheritence {
    private String rideName = "Bumble Bee";

    public static void main(String[] args) {
        ThemeParkRide ride = new ThemeParkRide();

        ride.startRide();
        System.out.println("Ride Name: " + ride.rideName);
    }
}

File: dartFiles/Inheritence.dart
class Ride {
  String category = "Roller Coasters";
  void startRide() {
    print("The ride has started!");
  }
}

class ThemeParkRide extends Ride {
  String _ridename = "Bumble Bee";
}

void main() {
  ThemeParkRide ride = ThemeParkRide();
  
  ride.startRide();
  print("Ride Name: ${ride._ridename}.");
}